qutebrowser/test/mainwindow/conftest.py

```diff
@@ -0,0 +1,34 @@
+"""
+pytest fixtures and utilities for testing.
+
+Fixtures defined here will be visible to all test files in this directory and
+below.
+"""
+
+import pytest
+
+from qutebrowser.config.config import ConfigManager
+from qutebrowser.utils import objreg
+
+
+@pytest.yield_fixture
+def default_config():
+    """
+    Fixture that registers an empty config object into the objreg module.
+
+    Should be used by tests which create widgets that obtain their initial state
+    from the global config object.
+
+    Note:
+
+        If we declare this fixture like this:
+
+            @pytest.yield_fixture(autouse=True)
+
+        Then all tests below this file will have a default config registered
+        and ready for use. Is that desirable?
```

Comment:
Hmm... if creating the `ConfigManager` object is cheap, we might just go and add the `autouse` declaration and let every user have an empty configuration registered to them (I would rename the fixture `empty_config` then). 

With that, the configuration can still be customized during the test function itself:

```python
def test_foo(empty_config):
     empty_config.set(...)
```

Or if one or more tests will share the same configuration, fixture can be created which will be used by those tests:

```python
@pytest.fixture
def dark_theme_config(empty_config):
     empty_config.set('colors', 'completion.bg', ...)
     ...
     return empty_config

def test_prompt_bg(dark_theme_config):
     ...

def test_prompt_fg(dark_theme_config):
     ...
```

As we can see, even if the fixture is declared `autouse`, we can still reuse them within other fixtures.

Anyway, this decision can be made later after we write more tests and have a better view of what is needed. :smile: